src/code/z_player_lib.c

```diff
@@ -1640,7 +1640,7 @@ void Player_PostLimbDrawGameplay(PlayState* play, s32 limbIndex, Gfx** dList, Ve
     }
 }
 
-u32 func_80091738(PlayState* play, u8* segment, SkelAnime* skelAnime) {
+u32 Player_InitDrawPause(PlayState* play, u8* segment, SkelAnime* skelAnime) {
```

Comment:
Wow, this is pretty juicy stuff.

If I'm understanding it right, `PAUSE_PLAYER_SEGMENT_GAMEPLAY_KEEP_BUFFER_SIZE` really stores how much of gameplay_keep to keep in memory. 0x5000 is only enough to load a bit past Link's animation tables, then the rest gets clobbered by the following DmaMgr_SendRequest1 call for the player object.

Not sure what `PAUSE_EQUIP_BUFFER` is suppose to be. Seems like it would be a framebuffer, but I'm having trouble finding where the memory is actually being used.

Reply:
Yeah `PAUSE_EQUIP_BUFFER_SIZE` is an image, it's the player prerender in the equip page of the pause menu

It is referenced here
```c
            PreRender_SetValuesSave(&sPlayerPreRender, PAUSE_EQUIP_PLAYER_WIDTH, PAUSE_EQUIP_PLAYER_HEIGHT,
                                    pauseCtx->playerSegment, NULL, sPreRenderCvg);
```

then `KaleidoScope_SetupPlayerPreRender` has `PreRender_SaveFramebuffer` to copy the drawn player (which is drawn in-between by `KaleidoScope_DrawPlayerWork`->`Player_DrawPause`->`Player_DrawPauseImpl`)

Note the player is first drawn to the current frame buffer, which is also partly used as a zbuffer (see `Player_DrawPause`), it's not directly drawn to the playerSegment